blocks/github-pr-overview/src/pull-request-selector/index.tsx

```diff
@@ -50,6 +50,10 @@ export const PullRequestSelector: FunctionComponent<
     return repoMap;
   }, [allPrs]);
 
+  if (!selectedRepository) {
+    return null;
+  }
```

Comment:
This looks like it would prevent anyone selecting a repository, given that the dropdown to select one is below

Reply:
Thanks for spotting this bug! It should be fixed in https://github.com/hashintel/hash/pull/1390/commits/d596ebe0a5b3ad71fecd0c7a9757e802fb952191.